### PR TITLE
Updated the website logo and added top 3 restaurant query 

### PR DESCRIPTION
**Pull request by kayheen**

### CODE REVIEW

**Comment by kayheen** [comment]
Possible implementations in future: 
1. Ratings can be in 1 dp instead of whole numbers to differentiate the different ratings between restaurants
2. Ratings can be more visual in terms of colouring of stars. 
3. To include the description and cuisine type of the Restaurant.

File: controllers/index.controller.js [hunk 1]
```diff
@@ -2,7 +2,11 @@ const db = require('../db/index');
 
 exports.showHomePage = async (req, res, next) => {
   try {
-    res.render('home');
+    const restaurant = await db.any('SELECT rname, round(AVG(rating)) AS avg FROM ReserveTimeSlots GROUP BY rname ORDER BY AVG(rating) DESC, rname LIMIT 3');
+    res.render('home', {
+      title: 'Ketchup and Chill!',
+      restaurants: restaurant
```
Comment (wailu) [inline controllers/index.controller.js]: Think will be better to call it `restaurants` here as there are multiple!

File: db/tables.sql [hunk 2]
```diff
@@ -121,4 +121,12 @@ CALL add_diner('wailunoob', 'Lim Wai Lun', 'wailunoob');
...
\ No newline at end of file
+CALL add_rowner('Dominoes', 'Domino', 'domiyess');
+
+-- Sample testing for Top 3 Restaurants --
+INSERT INTO Users(uname,name, pass) values  ('kh', 'kayheen', '123'), ('wy', 'wenyi', '456');
+INSERT INTO Diners values ('kh');
+INSERT INTO RestaurantOwners values ('wy');
+INSERT INTO OwnedRestaurants values ('wy', 'fudstreet', 'Popeyes'), ('wy', '1street', 'Bibimbap');
+INSERT INTO HasTimeSlots values ('19/10/2019', '10:00', 'Popeyes', 'fudstreet', 10), ('19/10/2019', '12:00', 'Popeyes', 'fudstreet', 10), ('19/10/2019', '14:00', 'Bibimbap', '1street', 10);
+INSERT INTO ReserveTimeSlots values ('19/10/2019', '10:00', 'Popeyes', 'fudstreet', 'kh', 'gr8', '3', '2', TRUE), ('19/10/2019', '12:00', 'Popeyes', 'fudstreet', 'kh', 'gr9', '4','3', TRUE), ('19/10/2019', '14:00', 'Bibimbap', '1street', 'kh', 'gr10', '5', '2', TRUE);
```
Comment (wailu) [inline db/tables.sql]: Really small thing but it's called `ReserveTimeslots` and `HasTimeslots` hahaha (doesn't affect anything but it'll look standardised!)

File: views/home.pug [hunk 3]
```diff
@@ -22,18 +22,21 @@ block content
...
-      + restaurant-card
-      + restaurant-card
-
+      each rest in restaurants
+        +restaurant-card(rest.rname, rest.avg)
+      else 
+        p No available top restaurants at the moment. Please check back later. 
+        //+restaurant-card
+        //+restaurant-card
+        //+restaurant-card
```
Comment (wailu) [inline views/home.pug]: Can delete already!

File: views/home.pug [hunk 4]
```diff
@@ -22,18 +22,21 @@ block content
...
       div.form-group
-        label restaurant
+        label Restaurant
         input.form-control(name='restaurant' type='text')
       button.btn.btn-light(type='submit') Search
   div#rest
-    h1 #[strong Top restaurants]
+    h1 #[strong Top Restaurants]
     div#top-restaurants
       // eventually fill the cards with data from model
```
Comment (wailu) [inline views/home.pug]: can delete the comment! :)

**Review by wailu — APPROVED** [review]
Left some comments! Overall looks good to go! 🤗 :100:

**Comment by wailu** [comment]
> Possible implementations in future:
> 
> 1. Ratings can be in 1 dp instead of whole numbers to differentiate the different ratings between restaurants
> 2. Ratings can be more visual in terms of colouring of stars.
> 3. To include the description and cuisine type of the Restaurant.

For point 2, @tehwenyi actually created the star look in the diner profile page! We could convert that into a mixin under `view/components`, then can use in both.

File: views/home.pug (hunk 4 above)
Comment (kayheen) [reply views/home.pug]: resolved!

**Pull request merged**